include/zephyr/drivers/dac.h

```diff
@@ -101,6 +101,11 @@ static inline int z_impl_dac_channel_setup(const struct device *dev,
 	return api->channel_setup(dev, channel_cfg);
 }
 
+/**
+ * @brief Broadcast channel number for DACs that support it.
+ */
+#define DAC_CHANNEL_BROADCAST	0xFF
+
```

Comment:
Sure can. I chose here because the broadcast channel ID only applies to the dac_write_value() API and not the dac_channel_setup() API in my implementation. If it is higher in the header file, there could be confusion where it could be used. If you have no similar concerns, I'll move it.

Reply:
I've added a note in the comment that it only applies to dac_write_value() and moved to the top of the file as suggested.